kroxylicious-api/src/main/java/io/kroxylicious/proxy/config/tls/SslProtocol.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.proxy.config.tls;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public enum SslProtocol {
+    SSLv2("SSLv2Hello"),
```

Comment:
Can you make the enums comply with [java:S115](https://next.sonarqube.com/sonarqube/coding_rules?open=java%3AS115&rule_key=java%3AS115)?